schema/ridl/_example/example0.ridl

```diff
@@ -15,14 +15,14 @@ enum Kind2: uint32
   - ADMIN                # aka, = 1
 
 
-message Empty
+struct Empty
```

Comment:
btw.. do you like the keyword `struct` ...? another idea is we could use the keyword here `type` ? it will mean inside of schema/ package the "Kind" will be "type" which maybe is confusing, but we could move/rename a few things. I'm also fine with `struct` as we have here

Reply:
I think `struct` is fine